Sources/AppStoreConnectCLI/Services/Operations/ListBundleIdsOperation.swift

```diff
@@ -33,6 +21,24 @@ struct ListBundleIdsOperation: APIOperation {
     typealias BundleId = AppStoreConnect_Swift_SDK.BundleId
 
     func execute(with requestor: EndpointRequestor) throws -> AnyPublisher<[BundleId], Error> {
-        requestor.request(options.endpoint).map(\.data).eraseToAnyPublisher()
+        let platforms = options.platforms.compactMap(Platform.init(rawValue:))
+
+        var filters: [BundleIds.Filter] = []
+        filters += options.identifiers.isEmpty ? [] : [.identifier(options.identifiers)]
```

Comment:
We should pick one way of doing this.  This is less "cute" than the alternative above. But then avoiding "cute"/"smart"/"tricky" code is preferred.